### ISSUE TITLE
Logic error when running Nova

### ISSUE THREAD

**Issue by Rayzr522** [NovaMods/nova-renderer#107]
Thanks to your amazing support, I have now gotten the project to complete both the `setup` and `buildDll` phases, so naturally I was eager to move on to `runNova` and behold the beauty that is the (incomplete) nova renderer... 

... only to have it crash :P

[Here's the console output](http://ix.io/EMW), but here's the relevant part:
 
```bash
$ ./gradlew runNova
...
[06:14:52] [Nova Renderer/INFO]: Loading complete
[06:14:52] [Nova Renderer/INFO]: Finished sending out initial config
[06:14:54] [Nova Renderer/INFO]: Adding texture gui (4096x2048)
terminate called after throwing an instance of 'std::logic_error'
  what():  basic_string::_M_construct null not valid
:runNova FAILED

```

The exact steps I took:

```bash
$ git clone https://github.com/NovaMods/nova-renderer -b vulkan
$ cd nova-renderer
$ ./gradlew setup
$ ./gradlew buildDll
$ ./gradlew runNova
```

If you need any more information, feel free to let me know, I'm happy to provide whatever you need.

**Comment by DethRaid** [comment]
Hey. Sorry for the long response time.

I'm running Nova on Linux and I've run into this exact error. I'm working on debugging it now

**Comment by DethRaid** [comment]
This should be fixed in commit 0e0ba72 in the `vulkan` branch (which is the current development branch) although there's a few additional errors cause I'm still developing the Vulkan backend

**Comment by Rayzr522** [comment]
Yep, all fixed. (Tho I'm having other errors, but I'll make a separate issue for those :P)
